_pages/welcome-to-TTS/getting-started.md

```diff
@@ -0,0 +1,255 @@
+[this lightweight primer](https://docs.google.com/presentation/d/1vXZDhEONp-h-zmyCt9wpwscoANouVK4fg5br6Ynzlpk/edit#slide=id.g54fbd835cd_0_25), 
+especially if you haven't worked on information security before. 
+If you're looking to go into greater detail, 
+18F's [Before You Ship](https://before-you-ship.18f.gov/) guide is definitive and should answer any questions you have, 
+but it can also be a little overwhelming. 
+If you ever need a quick answer, clarification or just general Security and Compliance help, 
+head over to [#wg-security](https://gsa-tts.slack.com/messages/CG8SNAEQM) 
+or [#wg-compliance](https://gsa-tts.slack.com/messages/C2UUQ5CNP) in Slack.
+
+If you hear "ATO" or "Authority to Operate" and are wondering what that's about, 
+[Before You Ship](https://before-you-ship.18f.gov/) and the Security and Compliance 
+leads can help you out there, too.
+
+You may also be wondering about security clearances. It's not something you need to worry about now.
+If you are in 18F and staffed to a project that needs clearance, 
+your account manager and staffing rep should make sure that any clearance needs are taken into account. 
+Still, it's never a bad idea to ask if you're wondering.
+
+
+## TTS Products and Platforms
+
+TTS offers many [products and platforms](https://docs.google.com/document/d/1vfBiQsprY1m8dG4tS_GeXkhG6ThF5ZqW9JKlDlvbWis/edit) to federal agencies, and we are strong proponents of using those products and platforms ourselves, too. Here are three TTS offerings that you will likely interact with early on (and often) during your tenure here.
+
+### Federalist
+
+It won't take long at TTS until you hear someone talking --- usually emphatically and ecstatically --- about [Federalist](https://federalist.18f.gov/), which is a really easy way to get a .gov site up and running. If you're already familiar with static site generators, then Federalist will already be pretty clear to you. If you're not, that's okay. It takes care of the hardest bits, and [each step you need to take is well-documented](https://federalist.18f.gov/documentation/). There's even [a handy video to help you get started](https://www.youtube.com/watch?v=OdhZz7udd1o). If you ever get stuck, give a wave in [#federalist](https://gsa-tts.slack.com/messages/C04823NMA) or [#federalist-support](https://gsa-tts.slack.com/messages/C1NUUGTT5) and they'll get you back on track.
+
+### Cloud.gov
+
+Just as Federalist makes creating .gov sites as easy as possible, [Cloud.gov ](https://cloud.gov/)makes hosting them as easy as possible, too, by dealing with a bunch of security and compliance sticking points so you don't have to. That convenience and trusted status means almost everything we build is hosted on cloud.gov. Like Federalist, [cloud.gov is extensively documented](https://cloud.gov/docs/) and heavily supported in [#cg-support](https://gsa-tts.slack.com/messages/C09CR1Q9Z). Still not clear? [They have a video, too](https://www.youtube.com/watch?v=nCycNkxs-rE).
```

Comment:
maybe "almost everything we build is hosted on cloud.gov, including Federalist."